akka-actor-tests/src/test/scala/akka/event/EventStreamSpec.scala

```diff
@@ -398,17 +397,17 @@ class EventStreamSpec extends AkkaSpec(EventStreamSpec.config) {
 
         es.subscribe(a2.ref, classOf[A])
         es.subscribe(a2.ref, classOf[T])
-        fishForDebugMessage(a1, s"watching ${a2.ref}", 1 second)
-        fishForDebugMessage(a1, s"watching ${a2.ref}", 1 second) // the unsubscriber "starts to watch" each time, as watching is idempotent
```

Comment:
This was the original failing test. remainingOrDefault (3 sec dilated) should work here.